Traceback (most recent call last):
  File "snippet.py", line 217, in <module>
    srcfn=sys.argv[1]
IndexError: list index out of range
